models/experimental.py

```diff
@@ -86,6 +88,39 @@ def forward(self, x, augment=False, profile=False, visualize=False):
         y = torch.cat(y, 1)  # nms ensemble
         return y, None  # inference, train output
 
+class Custom_Layer(nn.Module):
```

Comment:
Keep this class at the end of the file so that conflicts are avoided while pulling their changes